### PR TITLE
Fix rubocop after disabling BooleanSymbol cop

### PR DESCRIPTION
**Pull request by root-expert**

### CODE REVIEW

**Review by alexjfisher — CHANGES_REQUESTED** [review]
This doesn't revert the changes that were made to the types/providers.

Suggest doing a revert of https://github.com/voxpupuli/puppet-grafana/commit/70c2adc1abca80bf679f2cd87f72b34e578c912e and then rerunning the original autocorrect rake task.

**Comment by alexjfisher** [comment]
> Makes sense, Thanks!

Sorry, just realised you might not have had the full context, and only come across the issue because of the now failing build.
What we really need to fix is https://github.com/voxpupuli/puppet-grafana/issues/270#issuecomment-981929796

File: lib/puppet/provider/grafana_datasource/grafana.rb [hunk 1]
```diff
@@ -147,7 +147,7 @@ def password=(value)
     save_datasource
   end
 
-  # rubocop:disable Naming/PredicateName
+  # rubocop:disable Style/PredicateName
```
Comment (root-expert) [inline lib/puppet/provider/grafana_datasource/grafana.rb:150]: For these rubocop complains although autofix changed them?

File: lib/puppet/provider/grafana_dashboard_permission/grafana.rb [hunk 2]
```diff
@@ -218,7 +218,7 @@ def existing_permissions
     end
   end
 
-  def permission_data(destroy = false) # rubocop:disable Style/OptionalBooleanParameter
+  def permission_data(destroy = false)
```
Comment (root-expert) [inline lib/puppet/provider/grafana_dashboard_permission/grafana.rb]: Thoughts for this one?
I added `# rubocop:disable Style/OptionalBooleanParameter`

File: lib/puppet/provider/grafana_dashboard_permission/grafana.rb [hunk 3]
```diff
@@ -182,7 +182,7 @@ def permission=(value)
   def new_permission
     key = resource[:user] ? :userId : :teamId
     subject_id = resource[:user] ? user[:id] : team[:id]
-    permission = case resource[:permission] # rubocop:disable Style/HashLikeCase
+    permission = case resource[:permission]
```
Comment (root-expert) [inline lib/puppet/provider/grafana_dashboard_permission/grafana.rb]: And this one

File: lib/puppet/provider/grafana_datasource/grafana.rb (hunk 1 above)
Comment (bastelfreak) [reply lib/puppet/provider/grafana_datasource/grafana.rb:150]: what did it complain about? In an older rubocop version it was in the Naming namespace and the cop got moved to Style (or the other way around).

File: spec/classes/grafana_spec.rb [hunk 4]
```diff
@@ -183,7 +183,7 @@
           describe 'create data_dir' do
             it { is_expected.to contain_file('/var/lib/grafana').with_ensure('directory') }
           end
-        when 'FreBSD'
+        when 'FreeBSD'
```
Comment (bastelfreak) [inline spec/classes/grafana_spec.rb:186]: ah lol

File: lib/puppet/provider/grafana_datasource/grafana.rb (hunk 1 above)
Comment (root-expert) [reply lib/puppet/provider/grafana_datasource/grafana.rb:150]: https://github.com/voxpupuli/puppet-grafana/runs/4368546552?check_suite_focus=true

It's not erroring though

File: spec/classes/grafana_spec.rb (hunk 4 above)
Comment (root-expert) [reply spec/classes/grafana_spec.rb:186]: :smile:

**Review by alexjfisher — APPROVED** [review]

**Pull request merged**